frontend/src/stores/levels/actions.ts

```diff
@@ -13,4 +13,14 @@ async function getLevelsByRoleId(roleId: number): Promise<Level[] | undefined> {
   return role.levels
 }
 
-export { getLevelsByRoleId }
+async function removeLevel(levelId: number) {
+  await deleteRequest(`/levels/${levelId}`)
+}
+
+async function addLevel(level: Level, role: Role): Promise<void> {
+  const levelWithRoleId = { ...level, roleId: role.id }
+  await post<Level[]>('/levels', levelWithRoleId)
+  role.levels?.push(level)
+}
+
+export { getLevelsByRoleId, removeLevel, addLevel }
```

Comment:
```suggestion
export {removeLevel, addLevel }
```

Can you just put this method in getters file, instead of in actions?